entries/r/recruitee.com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Recruitee": {
+    "domain": "recruitee.com",
+    "url": "https://recruitee.com/",
```

Comment:
As the error message states, Defining the URL property isn't necessary when it's the same as the default value.
```suggestion
```